src/components/main/files/mod.rs

```diff
@@ -31,10 +35,59 @@ pub fn Files(cx: Scope<Props>) -> Element {
     let show_new_folder = use_state(&cx, || false);
     let show_upload = use_state(&cx, || false);
 
+    let file_storage = cx.props.storage.clone();
+
+    let root_directory = match file_storage.current_directory() {
+        Ok(current_directory) => current_directory, 
+        Err(error) => {
+            log::error!("Not possible to get root directory, error: {:?}", error);
+            Directory::default()
```

Comment:
Maybe use `Constellation::root_directory` instead of defaulting to `Directory::default`.